orm/model/ormtable/build.go

```diff
@@ -259,7 +260,7 @@ func Build(options Options) (Table, error) {
 			}
 		}
 
-		for name := range altNames {
+		for _, name := range maps.Keys(altNames) {
```

Comment:
This code isn't sorting the keys, so AFAICT doesn't produce behavior any different than ranging over the map directly?